rnd/autogpt_server/autogpt_server/data/execution.py

```diff
@@ -257,6 +257,20 @@ async def upsert_execution_output(
     )
 
 
+async def update_graph_execution_stats(graph_exec_id: str, stats: dict[str, Any]):
+    await AgentGraphExecution.prisma().update(
+        where={"id": graph_exec_id},
+        data={"stats": json.dumps(stats)},
+    )
+
+
+async def update_node_execution_stats(node_exec_id: str, stats: dict[str, Any]):
+    await AgentNodeExecution.prisma().update(
+        where={"id": node_exec_id},
+        data={"stats": json.dumps(stats)},
+    )
```

Comment:
It's good to first have this information in our DB, then I can have an ETL or another process that exports these values to sentry, somehow sentry (I have tried every version) causes a segfault when it's used in this project. I haven't got the chance to bisect which other component that doesn't work with this library, my random guess is cx_freeze (?)